pkg/openapiclient/local_files.go

```diff
@@ -65,7 +65,7 @@ func (k *localFilesClient) Paths() (map[string]openapi.GroupVersion, error) {
 		}
 
 		ext := strings.ToLower(filepath.Ext(f.Name()))
-		if ext != "json" && ext != "yml" && ext != "yaml" {
+		if ext != ".json" && ext != ".yml" && ext != ".yaml" {
```

Comment:
Should we use 

https://github.com/kubernetes-sigs/kubectl-validate/blob/27b2bf459cd8c677b4244cd954f87d66fc5102d8/pkg/utils/files.go#L18-L21